.github/workflows/main.yaml

```
@@ -98,3 +98,23 @@ jobs:
     - name: Run Style Check
       run: |
         PRE_COMMIT_HOME=$(pwd)/pre-commit-cache pre-commit run --all
+
+  twinecheck:
```

Can we add a `name` field here like?
```suggestion
  twinecheck:
      name: Validate PyPi dist with Twine
```
And while we're at it can we add names to the other jobs?